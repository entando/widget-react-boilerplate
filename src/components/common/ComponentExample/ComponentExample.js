import React from 'react';

export const ComponentExample = () => <div>This is an example</div>;

export default ComponentExample;
