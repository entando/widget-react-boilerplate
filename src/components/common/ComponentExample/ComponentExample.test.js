import React from 'react';
import { render } from '@testing-library/react';

import { ComponentExample } from './ComponentExample';

it('should render ComponentExample', () => {
  const { container } = render(<ComponentExample />);

  expect(container).toMatchSnapshot();
});
