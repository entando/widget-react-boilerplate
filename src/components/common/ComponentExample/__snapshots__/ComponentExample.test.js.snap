// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ComponentExample 1`] = `
<div>
  <div>
    This is an example
  </div>
</div>
`;
